datumaro/plugins/velodynepoints_format/extractor.py

```diff
@@ -0,0 +1,133 @@
+import os.path as osp
```

Comment:
There should be a license header with MIT license

Reply:
Resolved